### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by debjan**
Added trivial Python 3 support, and additionally:
- change `table` method to `tables` (following db.py method)
- replace `postgresql` uri scheme in docstring to `postgres` (former doesn't seem to work as db.py query template)

### CODE REVIEW

File: idb.py [hunk 1]
```diff
@@ -17,7 +20,8 @@ def check_db(func):
     @wraps(func)
     def deco(*args):
         if args[0]._db is None:
-            print '[ERROR]Please make connection: `con = %db_connect xx` or `%use_credentials xx` first!'  # noqa
+            print('[ERROR]Please make connection: `con = %db_connect xx` '
```
Comment (dongweiming) [inline idb.py:25]: i think `from __future__ import print_function` is necessary

Comment (debjan) [inline idb.py:25]: Ok, will do. Do you accept rename of `table` method?

Comment (dongweiming) [inline idb.py:25]: ok. i accept

Comment (debjan) [inline idb.py:25]: Done
Thanks

**Comment by dongweiming** [comment]
Thanks

**Pull request merged**